aiohttp/connector.py

```diff
@@ -446,6 +447,10 @@ def _close_immediately(self) -> List[Awaitable[object]]:
         finally:
             self._conns.clear()
             self._acquired.clear()
+            for keyed_waiters in self._waiters.values():
+                for keyed_waiter in keyed_waiters:
+                    if not keyed_waiter.done():
```

Comment:
I think this call is redundant. .cancel() already does that, if I remember correctly.

Reply:
agree
```
        if self._state != _PENDING:
            return False
```

Reply:
removed in 5a5260c99